packages/explorer/src/ui/explorer-ui-empty.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  Empty,
+  EmptyContent,
+  EmptyDescription,
+  EmptyHeader,
+  EmptyMedia,
+  EmptyTitle,
+} from '@workspace/ui/components/empty'
+import { LucideGlobe } from 'lucide-react'
+import type { ReactNode } from 'react'
+
+export function ExplorerUiEmpty({
```

Comment:
Once this is in, I'll refactor this into a generic `UiEmpty` component in `@workspace/ui` and replace other uses of `<Empty*>` from shadcn.